src/components/widgets/VoucherWidget.tsx

```diff
@@ -0,0 +1,43 @@
+import { useNavigation } from '@react-navigation/core';
+import React, { useCallback, useContext } from 'react';
+import { useQuery } from 'react-apollo';
+
+import { consts, Icon, normalize, texts } from '../../config';
+import { graphqlFetchPolicy } from '../../helpers';
+import { useRefreshTime } from '../../hooks';
+import { NetworkContext } from '../../NetworkProvider';
+import { getQuery, QUERY_TYPES } from '../../queries';
+import { ScreenName, WidgetProps } from '../../types';
+
+import { DefaultWidget } from './DefaultWidget';
+
+const { REFRESH_INTERVALS } = consts;
+
+export const VoucherWidget = ({ text }: WidgetProps) => {
+  const navigation = useNavigation();
+  const refreshTime = useRefreshTime('voucher-widget', REFRESH_INTERVALS.ONCE_A_DAY);
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+
+  const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp, refreshTime });
+
+  const { data, loading } = useQuery(getQuery(QUERY_TYPES.VOUCHERS), {
+    fetchPolicy,
+    skip: !refreshTime
+  });
+
+  // TODO: add redirection to Voucher IndexScreen
```

Comment:
really? i which one will be a vouchers index? the one with categories or the screen with "latest vouchers"? we do not have a screen with all vouchers, right?!